content/docs/started/kubeflow-overview.md

```diff
@@ -0,0 +1,151 @@
++++
+title = "Kubeflow Overview"
+description = "How Kubeflow helps you organize your ML workflow"
+weight = 10
++++
+
+<!--
+Note for authors: The source of the diagrams is held in Google Slides decks,
+in the "Doc diagrams" folder in the public Kubeflow shared drive.
+-->
+
+This guide introduces Kubeflow as a platform for developing and deploying a
+machine learning (ML) system.
+
+Kubeflow is a platform for data scientists who want to build and experiment with
+ML pipelines. Kubeflow is also for ML engineers and operational teams who want
+to deploy ML systems to various environments for development, testing, and
+production-level serving.
+
+## Architectural overview
+
+Kubeflow is *the ML toolkit for Kubernetes*.
+The following diagram shows Kubeflow as a platform for arranging the
+components of your ML system on top of Kubernetes:
+
+<img src="/docs/images/kubeflow-overview-platform-diagram.svg" 
+  alt="An architectural overview of Kubeflow on Kubernetes"
+  class="mt-3 mb-3 border border-info rounded">
+
+## Introducing the ML workflow
```

Comment:
What are the relative benefits of putting a section on the ML Workflow on the overview page vs. a separate page?

I think its helpful to orient people with some sort of diagram that shows how the tools fit together.

A couple questions

1. Is one such diagram going to be enough? 
1. Is it distracting/confusing to have steps in the diagram for which we don't have steps?

The approach we seem to be converging on is multiple diagrams corresponding to different CUJs.
Here's the diagram for the KF 1.0 CUJ
https://docs.google.com/presentation/d/1QR7-HGvzYiLzi-JHIntXs4whgpAhJY3VhRbgLpTwXas/edit#slide=id.g742bb082e3_0_391

Reply:
I like the [Kubeflow v1.0 CUJ diagram](https://docs.google.com/presentation/d/1QR7-HGvzYiLzi-JHIntXs4whgpAhJY3VhRbgLpTwXas/edit#slide=id.g742bb082e3_0_391)! One problem with it, though, is that it doesn't show the full capabilities of Kubeflow as a whole - it shows only the v1.0-stable components. It looks to me like this diagram belongs more in release notes and blog posts, rather than in the general docs.

On whether we should have the workflow diagrams on the overview page: I think the diagrams are an essential part of the overview, as they give people an immediate, visual way of seeing what Kubeflow can do for them.

That said, my goal is to keep the diagrams as simple as possible, so that people can quickly apply the workflow to their own specific needs rather than getting confused by bits that are too specific and therefore irrelevant. 

+1 to your comment that we may need more workflow diagrams, even per CUJ. They'd go in another section of the docs - the [use cases](https://www.kubeflow.org/docs/use-cases/) section is a good place. I plan to use this as a focus for the upcoming **doc sprint**.

Let me know if you have any ideas on simplifying the diagrams while still keeping the whole of Kubeflow represented. :)

Reply:
Sounds good.